src/lib.rs

```diff
@@ -166,7 +166,7 @@ extern crate proc_macro;
 #[proc_macro_derive(ConvertGraphQLEdgesNode, attributes(graphql_client_type))]
 pub fn derive_from_graphql_client_autogen(input: TokenStream) -> TokenStream {
     derive_from_graphql_client_autogen_2(input.into())
-        .unwrap()
+        .expect("valid input struct")
```

Comment:
Could you handle this with `panic!` macro, instead of `expect` please? The file lib.rs is the code for proc-macro , which runs against the Giganto codebase, during compile time only, to generate code from our custom macro. And this part can panic, when the Giganto codebase uses the custom macro incorrectly. My personal idea is that the message could be something like "ConvertGraphQLEdgesNode macro is not correctly used for {input}"